alinka/widget/containers/main_body/content/application/application_tabs/applicants_tab/__init__.py

```diff
@@ -48,3 +48,26 @@ def address_child_checkbox(self) -> bool:
     @property
     def address_first_parent_checkbox(self) -> bool:
         return self.applicant_2_data_group.address_checkbox.checkbox.isChecked()
+
+    def clear(self) -> None:
+        self.applicant_1_data_group.clear()
+        self.applicant_2_data_group.clear()
+        self.applicant_1_data_group.address_checkbox.checkbox.setChecked(False)
+        self.applicant_2_data_group.address_checkbox.checkbox.setChecked(False)
+
+    def populate_data(self, document_data: DocumentData) -> None:
+        applicants = document_data.applicants
+        checkbox1 = document_data.address_child_checkbox
+        checkbox2 = document_data.address_first_parent_checkbox
+
+        if len(applicants) == 1:
+            self.applicant_1_data_group.populate_applicant_data(applicants[0])
+            self.applicant_1_data_group.address_checkbox.checkbox.setChecked(checkbox1)
+            self.applicant_2_data_group.clear()
+            self.applicant_2_data_group.checkable = False
+            self.applicant_2_data_group.setFixedHeight(0)
```

Comment:
cannot we just `self.applicant_2_data_group.setVisible(False)` on this widget?

Reply:
Probably it takes space, even if it's not visible. 
But I have no idea what's was the purpose of this `.checkable`.... Let me.. hmm check it :)

Reply:
I'll remove it.